core/controllers/contributor_dashboard.py

```diff
@@ -260,7 +260,9 @@ def _content_in_review(self, state_name, content_id, suggestions):
         """
         return any(
             s.change.state_name == state_name and
-            s.change.content_id == content_id for s in suggestions)
+            s.change.content_id == content_id and
+            s.language_code == self.request.get('language_code')
```

Comment:
I think what we actually want to do here is to make sure when we fetch the suggestions here that we filter by the request `language_code` on line 210. Do you mind modifying the corresponding `suggestion_services.get_translation_suggestions_in_review_by_exploration` and `get_translation_suggestions_in_review_with_exp_id()` functions? They're not used anywhere else.

Reply:
Yup sure, I wasn't initially sure if the API was intentionally made to ignore language_code so went with the smallest diff. I'll update the methods accordingly!

Reply:
Done!